sources/it/provinces_v2.hjson

```diff
@@ -0,0 +1,227 @@
+{
+  versions: '>=8'
+  type: http
+  production: true
+  countryCode: IT
+  wikidata: Q15089
+  note: Italy provinces
+  data: https://sophox.org/regions/geojson.json
+  attribution: [
+    {
+      label: {
+        en: © OpenStreetMap contributors
+      }
+      url: {
+        en: https://www.openstreetmap.org/copyright
+      }
+    }
+    {
+      label: {
+        en: Elastic Maps Service
+      }
+      url: {
+        en: https://www.elastic.co/elastic-maps-service
+      }
+    }
+  ]
+  fieldMapping: [
+    {
+      type: id
+      name: iso_3166_2
+      desc: ISO-3166-2 identifier
+    }
+    {
+      type: id
+      name: istat
+      desc: ISTAT ID
+    }
+    {
+      type: property
+      name: label_en
+      desc: Province name (English)
+    }
+    {
+      type: property
+      name: label_it
+      desc: Province name (Italian)
+    }
+  ]
+  query: {
+    sparql:
+    '''
+    #defaultView:MapRegions
+    SELECT ?id ?iso_3166_2 ?istat ?label_en ?label_it
+    WHERE {
+      SERVICE <https://query.wikidata.org/sparql> {
+        {
+          SELECT
+            ?id
+            (SAMPLE(?id2) AS ?id2)
+            (SAMPLE(?iso_3166_2) AS ?iso_3166_2)
+            (SAMPLE(?istat) AS ?istat)
+            (SAMPLE(REPLACE(?label_en, "Province of ", "")) as ?label_en)
+            (SAMPLE(REPLACE(?label_it, "provincia di ", "")) as ?label_it)
```

Comment:
Is it possible to do multiple `REPLACE...` functions for each field? Maybe we can drop `Libero consorzio comunale di` from the communes.

Reply:
yep, it's possible it actually accepts a regex so not super beautiful but this made it

```sparql
(SAMPLE(REPLACE(?label_en, "(Province of |Metropolitan City of |Libero consorzio comunale di )", "")) as ?label_en)
(SAMPLE(REPLACE(?label_it, "((P|p)rovincia( autonoma|) (di |del |dell'|della )|città metropolitana di |Libero consorzio comunale di )", "")) as ?label_it)
```